lib/flipper/adapters/actor_limit.rb

```diff
@@ -0,0 +1,28 @@
+module Flipper
+  module Adapters
+    class ActorLimit < Wrapper
+      LimitExceeded = Class.new(Flipper::Error)
+
+      attr_reader :limit
+
+      def initialize(adapter, limit = 100)
+        super(adapter)
+        @limit = limit
+      end
+
+      def enable(feature, gate, resource)
+        if gate.is_a?(Flipper::Gates::Actor) && over_limit?(feature)
+          raise LimitExceeded, "Actor limit of #{@limit} exceeded for feature #{feature.key}"
```

Comment:
Should we encourage them to use groups or expressions and link to group enablement in the docs? 

Long term I think it would be cool to just have an entire page dedicated to this explaining why and how but a link should be enough for now.

Reply:
Added link to https://www.flippercloud.io/docs/features/actors#limitations (deploying now and should be available in a few minutes)